pkg/apis/externaldns/types.go

```diff
@@ -250,6 +253,7 @@ var defaultConfig = &Config{
 	TransIPAccountName:          "",
 	TransIPPrivateKeyFile:       "",
 	DigitalOceanAPIPageSize:     50,
+	ManagedDNSRecordTypes:       []string{endpoint.RecordTypeCNAME, endpoint.RecordTypeA},
```

Comment:
nit: maybe change the order of the record types like above? A and then CNAME?

Reply:
Looks good, I can make it consistent.